integreat_cms/api/v3/imprint.py

```diff
@@ -18,7 +19,8 @@ def transform_imprint(imprint_translation):
     """
     return {
         "id": imprint_translation.id,
-        "url": imprint_translation.get_absolute_url(),
+        "url": settings.BASE_URL + imprint_translation.get_absolute_url(),
+        "path": imprint_translation.get_absolute_url(),
```

Comment:
I believe `get_absolute_url()` needs a database request, so probably it should be executed only once and be stored to a variable?

Reply:
Okay... :joy: